Sources/SmokeDynamoDB/DynamoDBCompositePrimaryKeysProjection.swift

```diff
@@ -34,7 +34,8 @@ public protocol DynamoDBCompositePrimaryKeysProjection {
        the query.
      */
     func query<AttributesType>(forPartitionKey partitionKey: String,
-                               sortKeyCondition: AttributeCondition?)
```

Comment:
Again here, it doesn't make sense to provide the consistentRead parameter as an option. This is always going to be a GSI. ConsistentRead should always be false.

Reply:
Got it. Will make the change.